### DISCUSSION TITLE
Migration errors out after first creating a new project

### DISCUSSION THREAD

**Discussion by AmpGuy** [laravel/framework#36048] [Q&A]
What could be the cause of migration calls failing right off the bat in a newly created project "composer create-project laravel/laravel example-app" on Windows 10 Pro (new installation.) Turned off the firewall and still fails:
PDOException::("SQLSTATE[HY000] [2002] No connection could be made because the target machine actively refused it"

**Comment by malohtie** [comment]
your database credential are not correct i guess

**Comment by dyaskur** [comment]
Your DB_HOST is not valid, make sure you database(mysql) is running. if your DB is on local try to change `127.0.0.1` to `localhost`. sometime user not granted on `127.0.0.1` but granted on localhost.

**Comment by AmpGuy** [comment]
Thanks I found that mysql wasn't running. Once I caught it I was able to migrate. Thanks for your helpOn Jan 26, 2021 4:44 PM, Muhammad Dyas Yaskur <notifications@github.com> wrote:
Your DB_HOST is not valid, make sure you database(mysql) is running. if your DB is on local try to change 127.0.0.1 to localhost. sometime user not granted on 127.0.0.1 but granted on localhost.
